src/realm/set.hpp

```diff
@@ -665,13 +676,13 @@ REALM_NOINLINE auto Set<T>::find_impl(const T& value) const -> iterator
 template <class T>
 std::pair<size_t, bool> Set<T>::insert(T value)
 {
+    ensure_created();
```

Comment:
I think this makes the subsequent call to `update()` redundant right?

Reply:
Oh yes, my bad, I should have removed that.